guide/migration-from-riot2.md

```diff
@@ -41,12 +41,16 @@ Notice that now `data-is` could be used also in nested tags and it supports dyna
 ### Deprecate `name` and `id` to cache DOM nodes
 
 The attributes `name` and `id` will be no longer used by riot to cache DOM references to the tag context.
-Use the `ref` attribute instead
+Use the `ref` attribute instead.
+children tags are now targeted via `ref` too.
+This does not rename them in the `this.tags` object anymore, but adds them as tag instance to `this.refs`.
```

Comment:
Hmm, it doesn't make sense for me...
We have no change on `this.tags` spec, basically. How about "This does not scatter `this` name space anymore," or something.